lua/nvim-web-devicons/icons-default.lua

```diff
@@ -1136,6 +1136,12 @@ local icons_by_file_extension = {
     cterm_color = "28",
     name = "D",
   },
+  ["d.ts"] = {
+    icon = "󰛦",
+    color = "#519aba",
+    cterm_color = "74",
+    name = "TypeScript Declaration",
```

Comment:
```
E5248: Invalid character in group name
E5113: Error while calling lua chunk: ...pack/packer/start/chrisgrieser/lua/nvim-web-devicons.lua:296: Invalid highlight name: 'DevIconTypeScript Declaration'
stack traceback:
        [C]: in function 'nvim_set_hl'
        ...pack/packer/start/chrisgrieser/lua/nvim-web-devicons.lua:296: in function 'set_up_highlight'
        ...pack/packer/start/chrisgrieser/lua/nvim-web-devicons.lua:324: in function 'set_up_highlights'
        ...pack/packer/start/chrisgrieser/lua/nvim-web-devicons.lua:420: in function 'setup'
        ...d/site/pack/packer/start/nvim-tree.lua/lua/nvim-tree.lua:819: in function 'setup'
        /tmp/nd/nd.lua:86: in main chunk
```

vim can't have spaces in highlight group names

Reply:
fixed